lib/ndr_import/helpers/file/delimited.rb

```diff
@@ -37,6 +37,8 @@ def delimited_rows(path, col_sep = nil, liberal = false)
           # By now, we know `options` should let us read the whole
           # file succesfully; if there are problems, we should crash.
           CSV.foreach(safe_path, options.delete(:mode), **options) do |line|
+            # keeping last_data_column consistent with excel files where column numbers
+            # start at 1, delimited rows use an array index
```

Comment:
Does this comment still make sense here, given the new logic has moved into `Table`?

Reply:
Good spot, thanks @joshpencheon - sorted in 2670078